source/_posts/2022-09-07-release-20229.markdown

```diff
@@ -0,0 +1,420 @@
+of your automation you can do that via the overflow menu > Rename. Moving these items to the overflow ensures that the main editor page is used for the main functions of
+your automation.
+
+<img class="no-shadow" src='/images/blog/2022-09/automation_overflow.png' alt='Screenshot showing the new automation editor overflow menus'><br>
+
+Last, but definitely not least, Frenck added some long requested features to the automation forms. **State and Attribute value auto-completion!** Now if you are adding a
+state trigger, you can choose from a list of known states for that device type. These are also all translated into your language. Now there is no need to remember or find
+the states possible or format the state that needs to be entered. Just choose a state from the dropdown and let your worries wisp away!
+
+<img class="no-shadow" src='/images/blog/2022-09/automation_state_selector.gif' alt='Screenshot showing the new automation editor state and attribute value selector'><br>
+
+## New helper: Weekly schedule
+
+Have you ever wanted to run a schedule-based automation for the same time each week or day? Well, you are now in luck! With the addition of the Schedule helper, you are now able
+to easily create these schedules to build more consistent automations and timings.
+
+<img class="no-shadow" src='/images/blog/2022-09/create_schedule.gif' alt='A gif showcasing the new schedule helper and how to use the user interface'><br>
+
+{% my config_flow_start badge domain="schedule" %}
+
+## Bluetooth everywhere
+
+Last release we introduced the Bluetooth integration. This release, it's on fire 🔥 Not only do we now support multiple Bluetooth adapters, we also support ESPHome devices acting as [Bluetooth proxies](https://esphome.io/components/bluetooth_proxy.html) for your home. Wait what?!
+
+Yes 😎. You can now greatly expand the Bluetooth reach of Home Assistant using ordinary ESP32 devices. No extra hardware needed.
+
+<img class="no-shadow" src='/images/blog/2022-09/bluetooth-proxy-overview.png' alt='Picture of a home showing how Bluetooth data is proxied via ESPHome devices to Home Assistant'><br>
+
+To get a proxy up and running, head over to the [Bluetooth proxy installer website](https://esphome.github.io/bluetooth-proxies/) and install one straight from your browser.
+
+Bluetooth proxies are currently limited to forwarding passive data. Active connections are coming in a future release. Bluetooth integrations in Home Assistant will transparently use Bluetooth proxies without any extra coding!
+
+There is more! Besides 9 new supported brands, there is also support for a new open standard: [BTHome](https://bthome.io) by [@Ernst79]. It is automatically discovered by Home Assistant, devices can run over a year on a single battery and it supports data encryption.
+
+Thanks to [@bdraco], [@Ernst79], [@JesseRockz], [@jc2k] for all the hard, amazing and epic work 🙏.
+
+## Z-Wave Firmware updates now live
```

Comment:
```suggestion
## Z-Wave JS Firmware Update Service now live
```

Reply:
This is not about the service from my perspective, this is about HA offering updates from  our release notes perspective.

The suggestion is trying to make it about the backend, however,  these release notes are aimed towards changes in the Home Assistant itself.

Reply:
fair enough, I think I also misunderstood the intent of this - when we say updates are now live, I guess we are talking about the update entity specifically, not updates in general. Not sure of a way to make that more clear so maybe it's fine as is 👍🏾